konf/raw-site.yaml

```diff
@@ -0,0 +1,87 @@
+---
+
+kind: Service
+apiVersion: v1
+metadata:
+  name: ubuntu-com-security-api
+spec:
+  selector:
+    app: ubuntu-com-security-api
+  ports:
+    - name: http
+      protocol: TCP
+      port: 80
+      targetPort: http
+
+---
+
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: ubuntu-com-security-api
+spec:
+  replicas: 5
+  selector:
+    matchLabels:
+      app: ubuntu-com-security-api
+  template:
+    metadata:
+      labels:
+        app: ubuntu-com-security-api
+    spec:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                  - key: app
+                    operator: In
+                    values:
+                      - ubuntu-com-security-api
+              topologyKey: "kubernetes.io/hostname"
+      containers:
+        - name: ubuntu-com-security-api
+          image: prod-comms.docker-registry.canonical.com/ubuntu-com-security-api:latest
```

Comment:
Sorry, I forgot to mention this, here instead of `latest` use `${TAG_TO_DEPLOY}`

Reply:
Fixed!